.github/actions/automate-propagation/action.yml

```diff
@@ -22,12 +22,11 @@ runs:
     - name: Check is propagation PR
       id: check
       env:
-        EVENT_NAME: ${{ github.event_name }}
         LABEL: ${{ github.event.label.name }}
         PR_LOGIN: ${{ github.event.pull_request.user.login }}
       shell: bash
       run: |
-        if [[ $EVENT_NAME == 'labeled' && $LABEL == 'updatebot' && $PR_LOGIN == 'alfresco-build' ]]
+        if [[ $LABEL == 'updatebot' && $PR_LOGIN == 'alfresco-build' ]]
```

Comment:
@erdemedeiros wdyt of this change please?
When testing on a PR I noticed the event name was "pull_request" and this check was not on the original workfow (the workflow trigger was on this event type though)

Reply:
@atchertchian I don't fully understand your comment. The check was done globally [at the job level](https://github.com/Alfresco/alfresco-build-tools/pull/104/files#diff-407b2f2e7c243f844d95ba084196232884957fe2b3c90a9ac926efb25cfce721L18). Unfortunately it doesn't seem possible to have a global `if` at the action level.

Reply:
I had initially added this in the action, I will put it back globally on the caller workflow so we should be good.